WordPress/Classes/Categories/UIImageView+Gravatar.m

```diff
@@ -81,6 +83,13 @@ - (NSURL *)gravatarURLForEmail:(NSString *)email withSize:(NSInteger)size gravat
     return [NSURL URLWithString:gravatarUrl];
 }
 
+- (NSURL *)siteIconURLForSiteIconUrl:(NSString *)path
+{
+    NSInteger size = [self sizeForBlavatarDownload];
+    NSString *siteIconUrl = [NSString stringWithFormat:@"%@?w=%d&h=%d", path, size, size];
```

Comment:
The URL comes from the API, right? Even though I don't see any parameters in actual responses, I've seen code that sets `w=80`
https://github.com/Automattic/jetpack/blob/d5f87714a05fb3588dc8809600c29b5005abaadc/json-endpoints/class.wpcom-json-api-get-site-endpoint.php#L151

`NSURLComponents` might help in parsing the URL and checking for existing query parameters, and replacing them if they exist.